music_albums/music_album.rb

```diff
@@ -0,0 +1,26 @@
+require './item'
+
+class MusicAlbum < Item
+  attr_accessor :on_spotify, :publish_date, :archived, :title, :author, :genre, :label
+
+  def initialize(on_spotify, publish_date)
+    super(publish_date)
+    @on_spotify = on_spotify
+  end
+
+  def to_json(*_args)
+    "{
+      \"on_spotify\": \"#{on_spotify}\",
+      \"publish_date\": \"#{publish_date}\"
+    }"
+  end
+
+  # private
+
+  def can_be_archived?()
+    parent = super()
+    return true if parent || @on_spotify
```

Comment:
- [ ] `can_be_archived?` method should return true if the parent's method returns true **AND** if `@on_spotify` equals true, kindly pay attention here your using the `||` **OR** symbol. make sure to adjust your method so it returns the proper value according to the above condition